forward43/es_ingest.py

```diff
@@ -0,0 +1,36 @@
+import json
+import os
+
+from data.data_path import DATA_DIRECTORY
+
+import utils.elasticsearch as es_utils
+
+
+def load_to_es(filepath, es_object): 
+    '''
+    Load json data into elastic
+    @filepath  : json filepath which contains project details
+    @es_object : es instance
+    '''
+    try:
+        with open(filepath) as f:
+            data = json.load(f)
+
+        for document in data:
+            es_utils.index_document(es_object, 'forward43', '_doc', document, document['id'])
```

Comment:
I think this might break if the data is a line-delimited json: json doesn't handle a list of jsons very well :/ 
```
import json
with open("test.json", "w") as f:
    json_list = [{"a": "b"}, {"c": "d"}]
    json.dump(json_list, f, ensure_ascii=False, indent=4)
with open("test.json", "r") as f:
    data = json.load(f)
for point in data:
    print(point)
    print(point["b"])
```
gives:
```

{'a': 'b'}
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-32-532a067c8353> in <module>
      7 for point in data:
      8     print(point)
----> 9     print(point["b"])

KeyError: 'b'
```

I think your options are https://pypi.org/project/ndjson/ or doing a 
```
doc = json.loads(document)
es_utils.index_document(es_object, 'forward43', '_doc', doc, doc['id'])
```

Reply:
In the above example, it should be print(point["c"]) ;) 

ndjson is not required because the scrapers create a python list of dicts, which results in a valid json format
(webrobots.io uses ndjson format. I've written a helper function in utils/data.py to read data from that source)

Reply:
:3